pkg/workloads/cortex/downloader/download.py

```diff
@@ -32,14 +32,19 @@ def start(args):
         s3_client = S3(bucket_name, client_config={})
         s3_client.download(prefix, to_path)
         if args.unzip and os.path.basename(from_path).endswith("zip"):
-            util.extract_zip(os.path.join(to_path, os.path.basename(from_path)), delete_zip_file=True)
-
+            util.extract_zip(
+                os.path.join(to_path, os.path.basename(from_path)), delete_zip_file=True
+            )
 
 
 def main():
     parser = argparse.ArgumentParser()
     na = parser.add_argument_group("required named arguments")
-    na.add_argument("--download", required=True, help="comma separated list of path_to_download_from;path_to_download_to")
+    na.add_argument(
+        "--download",
+        required=True,
+        help="comma separated list of path_to_download_from;path_to_download_to",
```

Comment:
Can commas occur in s3 prefixes? Using the argpsrse append action mucky be safer: https://docs.python.org/2/library/argparse.html#action